mappings/net/minecraft/client/gui/screen/ingame/AbstractForgingScreen.mapping

```diff
@@ -1,10 +1,10 @@
-CLASS net/minecraft/class_4894 net/minecraft/client/gui/screen/ingame/ForgingScreen
-	FIELD field_22794 texture Lnet/minecraft/class_2960;
+CLASS net/minecraft/class_4894 net/minecraft/client/gui/screen/ingame/AbstractForgingScreen
```

Comment:
If you add `Abstract` to this screen, add `Abstract` to the handler as well.

Reply:
Where is the handler? o.O

Reply:
`ForgingScreenHandler` in `screen` package

Reply:
I wouldn't change it unless the handler class was also abstract, imo.

Reply:
maybe mojang just forgot to declare one of the classes abstract, who knows

Reply:
So, what's the decision on this?

Reply:
Probably needs a vote. I personally see no need to declare this abstract, as calling this `forging` already conveys abstraction.